UI/Pages/Accessibility.cshtml

```diff
@@ -4,6 +4,67 @@
+			limited number of users at the moment.
+		</p>
+		<p class="govuk-body">
+			We have taken steps to make this service accessible:
+		</p>
+		<ul class="govuk-list govuk-list--bullet govuk-!-margin-left-4">
+			<li>we designed it using the <a href="https://design-system.service.gov.uk/accessibility/"
+					class="govuk-link" data-testid="design-principles-link">GOV.UK accessible design principles</a></li>
+			<li>we did an audit to see if there were any accessibility issues</li>
+			<li>we tested the design with users with accessibility needs</li>
+		</ul>
+		<h1 class="govuk-heading-l" data-testid="access-parts-header">What to do if you cannot access parts of this website</h1>
+		<p class="govuk-body">
+			If you need information on this website in a different format, such as an accessible PDF, large print, easy
+			read, audio recording or braille, please email us on <a href="mailto:ntp.digital@education.gov.uk"
+				class="govuk-link" data-testid="mailto-ntp-link">ntp.digital@education.gov.uk</a>
 
-<p class="govuk-body">Accessibility statement</p>
\ No newline at end of file
+		</p>
+		<p class="govuk-body">
+			We’ll consider your request and respond to you within 5 working days.
+		</p>
+		<p class="govuk-body">
+			<a href="https://mcmw.abilitynet.org.uk/" class="govuk-link">AbilityNet</a> also has advice on making your
+			device easier to use if you’re disabled.
+		</p>
+		<h1 class="govuk-heading-l" data-testid="accessibility-reporting-header">Reporting accessibility problems with this website</h1>
+		<p class="govuk-body">
+			If you have any problems accessing the service or want to give us feedback, please email us on <a
+				href="mailto:ntp.digital@education.gov.uk" class="govuk-link">ntp.digital@education.gov.uk</a>
+		</p>
+		<h1 class="govuk-heading-l" data-testid="accessibility-enforcement-header">Enforcement procedure</h1>
+		<p class="govuk-body">
+			The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies
+			(Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’).
+		</p>
+		<p class="govuk-body">
+			If you’re not happy with how we respond to your complaint, please <a
+				href="https://www.equalityadvisoryservice.com/" class="govuk-link"
+				data-testid="contact-quality-link">contact the quality Advisory and Support
+				Service(EASS)</a>
+		</p>
+		<h1 class="govuk-heading-l" data-testid="accessibility-Technical-information-header">Technical information about
+			this website’s accessibility</h1>
+		<p class="govuk-body">
+			The Department for Education is committed to making its website services accessible, in accordance with the
+			Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+		</p>
+		<p class="govuk-body">
+			This website is or isn’t compliant with the <a href="https://www.w3.org/TR/WCAG21/" class="govuk-link">Web
```

Comment:
I think this sentence should pick one :)  either "is compliant" or "isn't compliant"